client/galaxy/scripts/mvc/ui/ui-misc.js

```diff
@@ -106,8 +106,8 @@ define(['utils/utils',
                     .addClass( this.model.get( 'cls' ) )
                     .attr( 'id', this.model.id )
                     .attr( 'type', this.model.get( 'type' ) )
-                    .attr( 'placeholder', this.model.get( 'placeholder' ) )
-                    .val( this.model.get( 'value' ) );
+                    .attr( 'placeholder', this.model.get( 'placeholder' ) );
+            this.model.get( 'value' ) !== this.$el.val() && this.$el.val( this.model.get( 'value' ) );
```

Comment:
This feels a bit cryptic to me `this.$el.val()` is used as an assignment here however its truthiness is being checked (I think). I would prefer explicit (even if longer).